src/derivatives/atlas.md

```diff
@@ -118,12 +115,12 @@ A guide for using macros can be found at
    "colin27-pipeline": {
       "sub-01": {
          "anat": {
-            "sub-01_res-1_label-brain_mask.nii.gz": "",
-            "sub-01_res-1_label-head_mask.nii.gz": "",
-            "sub-01_res-1_T1w.nii.gz": "",
-            "sub-01_res-1_T1w.json": "",
-            "sub-01_res-2_T1w.nii.gz": "",
-            "sub-01_res-2_T1w.json": "",
+            "atlas-colin27_res-1_seg-brain_mask.nii.gz": "",
```

Comment:
this is neither an atlas or template at this stage, but still sub-01. When aggregating across subjects, this will be a tpl and atlas (for segmentation).